wcc-contentful/lib/wcc/contentful/model_api.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+module WCC::Contentful::ModelAPI
+  extend ActiveSupport::Concern
+
+  included do
+    # We use a class var here because this is a global registry for all subclasses
+    # of this namespace
+    @@registry = {} # rubocop:disable Style/ClassVars
+  end
+
+  class_methods do
+    def schema(schema_json = nil, file: nil)
```

Comment:
Should a schema include an identifier (name, key, etc)? That way if you're trying to access models within a namespace / schema, that namespace / schema can be deterministically identified by something consistent within the code? We could derive from the contentful space, but that might change depending on environment OR cripple us if we wanted to do something cool like point at a copy space during a migration

IDK

Just thinking out loud